app/Models/Campaign.php

```diff
@@ -30,26 +30,25 @@ class Campaign extends Model
     use AutomateMutators;
     use BinHex;
 
-    /**
-     * The event map for the model.
-     *
-     * @var array
-     */
+    public static $rules = [
+    ];
+
+    protected $dates = [
+        'time_start',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "time_start" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)